pyemma/coordinates/clustering/include/bits/kmeans_bits.h

```diff
@@ -142,6 +142,36 @@ KMeans<dtype>::cluster(const np_array &np_chunk, const np_array &np_centers, int
     return return_new_centers;
 }
 
+
+template<typename dtype>
+std::tuple<py::array_t<dtype>, int, int> KMeans<dtype>::cluster_loop(const np_array& np_chunk, np_array& np_centers,
```

Comment:
you could typedef this `std::tuple<py::array_t<dtype>, int, int>` so it becomes more descriptive